src/aind_metadata_service/tars/query_builder.py

```diff
@@ -0,0 +1,44 @@
+"""Module that returns appropriate URL query strings"""
+from enum import Enum
+
+
+class APICalls(Enum):
+    """Enum of different API calls for TARS"""
+
+    REFERENCE_GENOMES = "api/v1/ReferenceGenomes"
+    TITER_IMPORTS = "/api/v1/TiterImports"
+    TITER_TYPES = "/api/v1/TiterTypes"
+    VIRAL_PREP_IMPORTS = "/api/v1/ViralPrepImports"
+    VIRAL_PREP_LOTS = "/api/v1/ViralPrepLots"
+    VIRAL_PREPS = "/api/v1/ViralPreps"
+    VIRAL_PREP_TYPES = "/api/v1/ViralPrepTypes"
+
+
+class URLQueries(Enum):
+    """Enum of queries"""
+
+    DEFAULT_ORDER = "order=1&orderBy=id"
+    SEARCH = "search"
+    SEARCH_FIELDS = "searchFields"
+
+
+class QuerySearchFieldValues(Enum):
+    """Enum of Search Field Values"""
+
+    LOT = "lot"
+
+
+class TarsQueries:
```

Comment:
It seems like your modelling this like querying from a SQL table. Since you're pulling info using a REST API, it should be designed differently.